src/ORM/DatabaseAdmin.php

```diff
@@ -73,6 +73,8 @@ protected function init()
             // "dev/tests" from CLI.
             || (Director::is_cli() && $allowAllCLI)
             || Permission::check("ADMIN")
+            || Permission::check("ALL_DEV_ADMIN")
+            || Permission::check("CAN_DEV_BUILD")
         );
         if (!$canAccess) {
```

Comment:
Please put this in a `canInit()` method to be consistent with the other dev controllers - especially since `DevelopmentAdmin` is looking for that method.

Reply:
Yeah this is interesting, this one.

This is not one of the controllers registered against `DevelopmentAdmin`, so it's not actually checked.

However, moving this to a `canInit()` at least makes it consistent and easy enough to move there if we need to

Reply:
Oohhhh good shout. Yeah it'd be good if when/if we register it everything's all in place and it "just works".